knowledge_repo/app/static/css/pages/base.css

```diff
@@ -31,6 +38,20 @@
         display: flex!important;
     }
 }
+
+/* Nav bar search form design */
+.form-control {
+  border-radius: 21px;
+  box-shadow: 0 0px 0px rgba(0,0,0,0);
+  border: 1px solid #e2ddddcf;
+  padding: 8px 14px;
+}
+
+.btn {
+  border-radius: 21px;
+  font-weight: 450;
```

Comment:
`450` probably isn't a real font weight here (ie the font files likely don't define a `450`). You probably want `400` or `500`

Reply:
👍